sqlalchemy_redshift/dialect.py

```diff
@@ -208,6 +208,15 @@ def __str__(self):
         else:
             return self.schema + "." + self.name
 
+    @classmethod
+    def _unquote(cls, part):
```

Comment:
```suggestion
    @staticmethod
    def _unquote(part):
```

I had asked for this to be a class method, but seeing it written out, I realize it should be `staticmethod` instead. Apologies for the churn.